doc/ref/gappkg.xml

```diff
@@ -173,5 +184,2074 @@ users.
+create <C>manual.lab</C> files yourself.
+<P/>
+To produce an HTML version of the manual one needs the Perl 5 program
+<F>convert.pl</F>. Instructions on its usage are contained as comments
+in this file.
+<P/>
+Finally, to ensure the mathematical formulae are rendered as well as they
+can be in the HTML version,  one  should  also  have  the  program  <C>tth</C>
+(&TeX;  to  HTML  converter);  <F>convert.pl</F>  calls  <F>tth</F>  to  translate
+mathmode formulae to HTML (if it's available). The <F>tth</F> program is  easy
+to compile and can be obtained from
+<URL>http://hutchinson.belmont.ma.us/tth/tth-noncom/download.html</URL>.
+<!--
+A prototype of the command to execute is in the file <F>make_doc</F>; 
+note that the HTML manual is produced in files with <C>.htm</C> extensions 
+in a directory <F>htm</F> outside the <F>doc</F> directory. 
+-->
+<P/>
+
+</Section>
+
+
+<!-- %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% -->
+<Section Label="An Example of a GAP Package">
+<Heading>An Example of a GAP Package</Heading>
+
+We illustrate the creation of a &GAP; package by an example of a very basic package. 
+<P/>
+Create the following directories in your home directory: 
+<F>.gap</F>, <F>.gap/pkg</F> and <F>.gap/pkg/test</F>.
+Then inside the directory <F>.gap/pkg/test</F> create an empty file 
+<F>init.g</F>, and a file <F>PackageInfo.g</F> with the following contents:
+<P/>
+<Log><![CDATA[
+SetPackageInfo( rec(
+  PackageName := "test",
+  Version := "1.0",
+  PackageDoc := rec(
+      BookName  := "test",
+      SixFile   := "doc/manual.six",
+      Autoload  := true ),
+  Dependencies := rec(
+      GAP       := "4.9",
+      NeededOtherPackages := [ ["GAPDoc", "1.6"] ],
+      SuggestedOtherPackages := [ ] ),
+  AvailabilityTest := ReturnTrue ) );
+]]></Log>
+<P/>
+This file  declares the &GAP; package with name <Q>test</Q> in version 1.0. 
+The package documentation consists of one autoloaded book; the <C>SixFile</C> 
+component is needed by the &GAP; help system. Package dependencies require at
+least &GAP;&nbsp;4.9 and &GAPDoc; package at version at least 1.6, and these
+conditions will be checked when the package will be loaded 
+(see <Ref Sect="Version Numbers"/>). 
+Since there are no requirements that have to be tested, 
+<C>AvailabilityTest</C> just uses <Ref Func="ReturnTrue"/>.
+<P/>
+Now start &GAP; (without using the <C>-r</C> option) and the <F>.gap</F>
+directory will be added to the &GAP; root directory to
+allow &GAP; to find the packages installed there
+(see <Ref Sect="GAP Root Directories"/>).
+<P/>
+<Log><![CDATA[
+gap> LoadPackage("test");
+true
+]]></Log>
+<P/>
+This &GAP; package is too simple  to be useful, but we have succeeded
+in loading it via <Ref Func="LoadPackage"/>, satisfying 
+all specified dependencies.
+<P/>
+You can also try to create a basic &GAP; package with the help of the
+tool called <Package>PackageMaker</Package>, created by Max Horn and
```

Comment:
It is my hope that others will help out with that tool, so perhaps don't bring up my name with that? The manual doesn't mention Max N. and Frank for GAPDoc all the time either, I think.